src/mesh/difops.cxx

```diff
@@ -167,15 +167,15 @@ const Field3D Grad_parP(const Field3D &apar, const Field3D &f) {
 * vparallel times the parallel derivative along unperturbed B-field
 *******************************************************************************/
 
-const Field2D Vpar_Grad_par(const Field2D &v, const Field2D &f, const CELL_LOC outloc) {
-  return f.getCoordinates(outloc)->Vpar_Grad_par(v, f, outloc);
+const Field2D Vpar_Grad_par(const Field2D &v, const Field2D &f, const CELL_LOC outloc, const DIFF_METHOD method) {
```

Comment:
I'm not sure why `test-code-style` doesn't catch this (and it's not new code) but usually we don't allow (for style reasons) the `const` qualifier on things that are passed by value. I've probably just misremembered/interpreted the style rules.

Reply:
I thought it looked unusual, but didn't know why... I'll make a commit that gets rid of those const's.